src/main/java/uk/ac/cam/cl/dtg/segue/dao/content/ContentMapper.java

```diff
@@ -82,17 +83,17 @@ public ContentMapper() {
   /**
    * Alternative constructor that will attempt to search for valid types to pre-register.
    *
-   * @param configuredReflectionClass - string representing the parent package to search for content classes.
-   *                                        e.g. uk.ac.cam.cl.dtg.segue
+   * @param classes - list of classes contained within the parent package to search for content classes.
```

Comment:
This is a bit nitpicky, but now that you've declared `classes` as a set, rather than "list" you could say "set". In fact, if you want to be super accurate, you could say "series" instead, and change the `classes` parameter to a `Collection`, because in this instance you don't really care what the exact type of `classes` is. Whatever it is, you will be converting it to a `Set` in line 95, when you do the filtering.